modules/dnn/include/opencv2/dnn/all_layers.hpp

```diff
@@ -648,6 +648,62 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
         static Ptr<Layer> create(const LayerParams& params);
     };
 
+    class CV_EXPORTS CopyLayer : public Layer
+    {
+        public:
+
+            String intype, outtype;
+            bool forceCopy, dontCast;
+
+            static Ptr<Layer> create(const LayerParams& params);
+    };
```

Comment:
I think that Copy can be mapped to something that already exists (Identity in example)

Reply:
Sure, I will try to find an alternative for it and will make a fix there.

Reply:
I looked at all the layers currently present in `layers/` and observed that `Identity` layer is not present. In addition, the implementation [here](https://github.com/torch/nn/blob/master/Copy.lua) suggests that it casts the input from `intype` to `outtype`, may be identity will not be able to do that from what I can figure out from the implementation of Identity layer available [here](https://github.com/torch/nn/blob/master/Identity.lua).
Please let me know your views.

Reply:
There is:

https://github.com/opencv/opencv/blob/4e3c09b4766d14569ae5ef75e52e57d3c18416cb/modules/dnn/src/init.cpp#L115

By default, OpenCV work with FP32 tensors data

Reply:
May be then mapping Copy to Identity will be a nice option. Thanks for the help. :-)

Reply:
It works for the model shared in the issue. Resolving.

Reply:
For most use cases it will work, though I am not sure, what will happen if, the `intype` and `outtype` are different, may be something like, `intype` is FP and `outtype` is `int`, where `Copy` will be helpful to truncate the decimal points.